docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -449,8 +449,10 @@ appropriate arguments.
 
 ## Upgrading
 
-Instructions for upgrading kubeadm clusters can be found
-[here](/docs/tasks/administer-cluster/kubeadm-upgrade-1-7/).
+Instructions for upgrading kubeadm clusters are available for:
+
+ * [1.7.x->1.7.y upgrades](/docs/tasks/administer-cluster/kubeadm-upgrade-1-7/)
+ * [1.7.x->1.8.y and 1.8.x->1.8.y upgrades](/docs/tasks/administer-cluster/kubeadm-upgrade-1-8/)
```

Comment:
All other; 1.7->1.7, 1.7->1.8, 1.8->1.8 is covered by kubeadm upgrade. You can split these cases in separate list items